.travis.yml

```diff
@@ -5,6 +5,9 @@ services:
 
 before_install:
   - docker build -t yast-s390-image .
+  # list the installed packages (just for easier debugging)
+  - docker run --rm -it yast-s390-image rpm -qa | sort
```

Comment:
`--rm` removes the result of the command, `rpm -qa` does not have any side effects so it's not strictly needed. But'd keep it for consistency, if we add something more complex later then we do not forget about it.